### DISCUSSION TITLE
[ISSUE] Please wait while the application is re-connecting...

### DISCUSSION THREAD

**Discussion by kitoheedong** [flet-dev/flet#3227] [Q&A]
I made flet based GUI program for windows local application.

[code for running the flet]
if __name__ == '__main__':
    flet.app(target=main, assets_dir="assets")

Everythings working fine at the beginning. However, at some point, it shows below window and stops working.
![image](https://github.com/flet-dev/flet/assets/109059327/79f24a8f-4c94-4b31-8c99-477d46c44078)

[versions]
python version: 3.7
flet version: 0.7.4

What am I supposed to do?

**Comment by bitmvk** [comment]
first, try updating your Flet package to the latest version using `pip install --upgrade flet` and try again
